From 9f7a061a593bc24e1e76146d96949e51a2f08595 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Thu, 17 May 2018 18:20:05 +0200
Subject: [PATCH 1/1] efi_selftest: do not call efi_init_obj_list() twice

Remove a superfluous call to efi_init_obj_list() invoked by
'bootefi selftest'.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 cmd/bootefi.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/cmd/bootefi.c b/cmd/bootefi.c
index fe50bac1c50..806339823ff 100644
--- a/cmd/bootefi.c
+++ b/cmd/bootefi.c
@@ -441,8 +441,6 @@ static int do_bootefi(cmd_tbl_t *cmdtp, int flag, int argc, char * const argv[])
 		 * callback entry
 		 */
 		efi_save_gd();
-		/* Initialize and populate EFI object list */
-		efi_init_obj_list();
 		/* Transfer environment variable efi_selftest as load options */
 		set_load_options(&loaded_image_info, "efi_selftest");
 		/* Execute the test */
-- 
2.14.2

